justice/datasets/plasticc_data.py

```diff
@@ -133,13 +134,64 @@ def _sqlite_get_lc(cls, conn, dataset, obj_id):
         lc.meta = dict(zip(cls.metadata_keys, meta_row))
         return lc
 
+    @classmethod
+    def _bcolz_get_lcs(cls, bcolz_table, meta_table, obj_ids):
```

Comment:
Add a small docstring saying what bcolz_table and meta_table are (both bcolz.ctable instances, right?)